types/inlineCompletionWithReferences.ts

```diff
@@ -9,6 +9,22 @@ export type InlineCompletionItemWithReferences = InlineCompletionItem & {
      */
     itemId: string
 
+    /**
+     * Flag to indicate this is an edit suggestion rather than a standard inline completion.
+     */
+    isInlineEdit?: boolean
```

Comment:
Will we support 1 inline completion and another prediction within the same client? If no, this flag is not needed as client already knows if it enabled edit predictions or no.